src/comparison.ts

```diff
@@ -0,0 +1,74 @@
+import deepdash from "deepdash"
+import lodash from "lodash"
+
+import { normalizeValue } from "./normalization"
+
+export const ld = deepdash(lodash)
+
+export const doValuesContainSameData = (v1: unknown, v2: unknown) => {
```

Comment:
- would be useful to have comments why/when you need this method 
- no return type - would be great to specify it, as it helps during reading of the code which wasn't written by you (or it was but long time ago) to not scroll back & forth to "interpret" by eyes the return type
- "normalizeValue" - looks like some magic is happening there.. 

side question: you considered to use [_.isEqual(object1, object2)](https://lodash.com/docs/4.17.15#isEqual) or [isDeepStrictEqual(object1, object2)](https://nodejs.org/api/util.html#util_util_isdeepstrictequal_val1_val2), and if yes => why not to use them? I assume that this method might work somehow differently, but would be good to learn why (may be in comments)

Reply:
> would be useful to have comments why/when you need this method
> side question: you considered to use [_.isEqual(object1, object2)](https://lodash.com/docs/4.17.15#isEqual) or [isDeepStrictEqual(object1, object2)](https://nodejs.org/api/util.html#util_util_isdeepstrictequal_val1_val2), and if yes => why not to use them? I assume that this method might work somehow differently, but would be good to learn why (may be in comments)

https://github.com/paritytech/opstooling-js/pull/4/commits/75fcf61a1a1ef50e875c8b987b12f4a38fb34408

> "normalizeValue" - looks like some magic is happening there..

https://github.com/paritytech/opstooling-js/pull/4/commits/4922d8c8a931bcccc24ccedac8b61a57d6ffa029

> no return type - would be great to specify it, as it helps during reading of the code which wasn't written by you (or it was but long time ago)

This reasoning should apply to the whole code. It makes sense to enable `@typescript-eslint/explicit-module-boundary-types` if that aspect is desired, even so that we don't waste time on reviews asking for this.

Reply:
> It makes sense to enable `@typescript-eslint/explicit-module-boundary-types` if that aspect is desired, even so that we don't waste time on reviews asking for this.

I'd vote for it

Reply:
https://github.com/paritytech/opstooling-js-style/issues/1